pytentiostat/config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "Experiment":{
+      "Type":"LSV"
+    },
+    "Plot":{
+      "Type":"plot"
+    },
+    "Write":{
+      "Type":"write"
+
+    },
+    "LSV":{
+        "Max_Volage":255,
```

Comment:
voltage?

Also, to avoid headaches later I would make all these all lower case, so ``max_voltage``, and so on.